Update HTTP Cache-Control s-maxage (#30985)
mdn/content: files/en-us/web/http/headers/cache-control/index.md

#### `s-maxage`

> The `s-maxage` response directive also indicates how long the response is [fresh](/en-US/docs/Web/HTTP/Caching#fresh_and_stale_based_on_age) for (similar to `max-age`) — but it is specific to shared caches and is always ignored by private caches. The `s-maxage` directive overrides the value specified by the `max-age` directive or the `Expires` header if they are present.

Reviewer: Great catch. I'm going to slightly reword this to make it more "stand alone" (less referring to `maxage` as a comparison).

```suggestion
The `s-maxage` response directive indicates how long the response remains [fresh](/en-US/docs/Web/HTTP/Caching#fresh_and_stale_based_on_age) in a shared cache.
The `s-maxage` directive is ignored by private caches, and overrides the value specified by the `max-age` directive or the `Expires` header for shared caches, if they are present.
```
